packages/react/src/components/Callout/Callout.types.ts

```diff
@@ -87,23 +87,23 @@ export interface ICalloutProps extends React.HTMLAttributes<HTMLDivElement>, Rea
   isBeakVisible?: boolean;
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33493&commitId=842f556de7bdbc77c499f321aadbd83837c180cb&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Bottom left edge.chromium.png|2309| Changed|
|Callout.Bottom Left Edge RTL.chromium.png|2178| Changed|
|Callout.Bottom Right Edge RTL.chromium.png|1593| Changed|
|Callout.No callout width specified.chromium.png|2319| Changed|
|Callout.Bottom center.chromium.png|2146| Changed|

</details>


  <details><summary><b>react-charting-HeatMapChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-HeatMapChart.Basic.chromium.png|376| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>